plugins/unescaped-markup/index.html

```diff
@@ -24,15 +24,23 @@ <h1>How to use</h1>
 	<p>This plugin provides several methods of achieving the same thing:</p>
 
 	<ul>
-		<li>Instead of using <code>&lt;pre>&lt;code></code> elements, use <code>&lt;script type="text/plain"></code>
-		<pre><code>&lt;script type="text/plain" class="language-markup">
+		<li>
+			Instead of using <code>&lt;pre>&lt;code></code> elements, use <code>&lt;script type="text/plain"></code>:
+
+			<pre><code>&lt;script type="text/plain" class="language-markup">
 &lt;p>Example&lt;/p>
 &lt;/script></code></pre>
 		</li>
-		<li>Use a HTML-comment to escape your code
-		<pre><code>&lt;pre class="language-markup">&lt;code>&lt;!--
+		<li>
+			Use a HTML-comment to escape your code:
```

Comment:
English is weird, but I think this is correct ("aych" (H) sound is a vowel sound).

```suggestion
			Use an HTML-comment to escape your code:
```